### DISCUSSION TITLE
[Script request] Jitsi

### DISCUSSION THREAD

**Discussion by 391a** [community-scripts/ProxmoxVE#974] [Request script]
### Application Name

 Jitsi

### Website

https://jitsi.org/

### Description

Jitsi Meet - Secure, Simple and Scalable Video Conferences that you use as a standalone app or embed in your web application.
https://jitsi.github.io/handbook/docs/intro/
https://github.com/jitsi

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by samwathegreat** [comment]
I second this request.  OP, please add [Script Request] flair to your title.

**Comment by akaw80** [comment]
I also would love to see this as an LXC for ease of install and use.

**Comment by Grelo4ka** [comment]
Upvoted!

**Comment by albotroz** [comment]
I have seen that Jitsi Turn-Server is recommended for smooth work.
It would therefore be advisable to offer a separate Proxmox script beforehand in order to run Jitsi on top of it in separate Container or VM, for example.

https://github.com/community-scripts/ProxmoxVE/discussions/3081

**Comment by kiwinh0** [comment]
Another interested in this great idea. It would be nice to have a dedicated Jitsi server.

**Comment by tremor021** [comment]
Seems simple enough

![image](https://github.com/user-attachments/assets/13ea4117-a8aa-409d-877c-9fff37cf5405)

**Comment by XavierBit** [comment]
Just here to second this idea! It would be wonderful to have a script for Jitsi!

**Comment by MickLesk** [comment]
https://github.com/community-scripts/ProxmoxVED/issues/1618
